packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx

```diff
@@ -90,13 +91,21 @@ export class SuggestionsControl<T> extends BaseComponent<ISuggestionsControlProp
   }
 
   public render(): JSX.Element {
-    const { className, headerItemsProps, footerItemsProps } = this.props;
+    const { className, headerItemsProps, footerItemsProps, suggestionsAvailableAlertText } = this.props;
 
+    const screenReaderText = mergeStyles(hiddenContentStyle);
     return (
       <div className={css('ms-Suggestions', className ? className : '', styles.root)}>
         {headerItemsProps && this.renderHeaderItems()}
         {this._renderSuggestions()}
         {footerItemsProps && this.renderFooterItems()}
+        {
```

Comment:
I don't think you need this enclosing `{`, since the expression is just a JSX element

Reply:
Actually I think it makes sense to move the conditional check to this level -- we don't want to display an empty aria alert, right?